SPECS/libxfce4util/libxfce4util.spec

```diff
@@ -1,8 +1,8 @@
-%global xfceversion 4.14
+%global xfceversion 4.19
```

Comment:
It might be better to automate this. Something like:
%define xfceversion %(echo %{version} | cut -d. -f1-2)